frontend/src/components/SocialMediaButton/index.js

```diff
@@ -65,4 +48,8 @@ const socialMediaButton = ({ name, username }) => {
   );
 };
 
-export default socialMediaButton;
+SocialMediaButton.propTypes = {
+  name: PropTypes.string.isRequired,
```

Comment:
Since this is a know list of options, what about using PropTypes.OneOf? Maybe with [Object.keys](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)?
```suggestion
  name: PropTypes.OneOf(Object.keys(SOCIALS)).isRequired,
```